lib/internal/process/pre_execution.js

```diff
@@ -76,15 +76,21 @@ function prepareShadowRealmExecution() {
 
   // Disable custom loaders in ShadowRealm.
   setupUserModules(true);
-  registerRealm(globalThis, {
```

Comment:
What happened to `registerRealm`? Does this comment need updating?

Reply:
Comments are on L88-L91